src/app/component-library/catalog/asset-detail-dialog/asset-property-grid-group-builder.ts

```diff
@@ -236,63 +236,70 @@ export class AssetPropertyGridGroupBuilder {
-            this.propertyGridUtils.formatDate(
-              contractAgreement.contractSigningDate,
-            ),
-          ),
-        },
-        {
-          icon: 'category',
-          label: 'Valid From',
-          ...this.propertyGridUtils.guessValue(
-            this.propertyGridUtils.formatDate(
-              contractAgreement.contractStartDate,
-            ),
-          ),
-        },
-        {
-          icon: 'category',
-          label: 'Valid To',
-          ...this.propertyGridUtils.guessValue(
-            this.propertyGridUtils.formatDate(
-              contractAgreement.contractEndDate,
-            ),
-          ),
-        },
-        {
-          icon: 'policy',
-          label: 'Direction',
-          ...this.propertyGridUtils.guessValue(contractAgreement.direction),
-        },
-        {
-          icon: 'link',
-          label: 'Other Connector Endpoint',
-          ...this.propertyGridUtils.guessValue(
-            contractAgreement.counterPartyAddress,
-          ),
-        },
-        {
-          icon: 'link',
-          label: 'Other Connector ID',
-          ...this.propertyGridUtils.guessValue(
-            contractAgreement.counterPartyId,
+    let properties: PropertyGridField[] = [];
+    properties = [
+      {
+        icon: 'category',
+        label: 'Signed',
+        ...this.propertyGridUtils.guessValue(
+          this.propertyGridUtils.formatDate(
+            contractAgreement.contractSigningDate,
           ),
-        },
-        {
-          icon: 'category',
-          label: 'Contract Agreement ID',
-          ...this.propertyGridUtils.guessValue(
-            contractAgreement.contractAgreementId,
+        ),
+      },
+      {
+        icon: 'category',
+        label: 'Valid From',
+        ...this.propertyGridUtils.guessValue(
+          this.propertyGridUtils.formatDate(
+            contractAgreement.contractStartDate,
           ),
-        },
-      ],
+        ),
+      },
+      {
+        icon: 'category',
+        label: 'Valid To',
+        ...this.propertyGridUtils.guessValue(
+          this.propertyGridUtils.formatDate(contractAgreement.contractEndDate),
+        ),
+      },
+      {
+        icon: 'policy',
+        label: 'Direction',
+        ...this.propertyGridUtils.guessValue(contractAgreement.direction),
+      },
+      {
+        icon: 'link',
+        label: 'Other Connector Endpoint',
+        ...this.propertyGridUtils.guessValue(
+          contractAgreement.counterPartyAddress,
+        ),
+      },
+      {
+        icon: 'link',
+        label: 'Other Connector ID',
+        ...this.propertyGridUtils.guessValue(contractAgreement.counterPartyId),
+      },
+      {
+        icon: 'category',
+        label: 'Contract Agreement ID',
+        ...this.propertyGridUtils.guessValue(
+          contractAgreement.contractAgreementId,
+        ),
+      },
+    ];
+    if (contractAgreement.isConsumingLimitsEnforced) {
```

Comment:
yes, very nice, exactly like this.

since we are doing another round with the PR anyway, let's also add the blank line between 290/291 as we have one betwen 298/300